Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+        - 6windgate-tools-common-libs-libconsole
+        - 6windgate-tools-common-libs-pyroute2
+        - 6wind-openstack-extensions
+        - dkms
+        - nuage-metadata-agent (6wind version)
+        - nuage-openvswitch (6wind version)
+        - python-pyelftools
+        - selinux-policy-nuage-avrs
+        - virtual-accelerator-base
+
+
+* Adding Nuage Heat Templates ( `nuage-tripleo-heat-templates <../../nuage-tripleo-heat-templates>`_  )
+
+    - Nuage provides heat templates & environment files to configure neutron on controller and nuage-openvswitch & nuage-metadata-agent on compute nodes and we configure VA on computeavrs nodes.
+
+* Updating the TripleO Heat templates (also referred to as the puppet manifests)
+
+    - Some of the parameters in ``neutron.conf`` and ``nova.conf`` need to be configured in the Heat templates. The Nuage VRS and metadata agent also need to be configured. The values for these parameters depend on the Nuage VSP configuration.
+      We use ``neutron-nuage-config.yaml`` and ``nova-nuage-config.yaml`` environment files to configure these values.
+    - See the `Sample Templates`_ section for some probable values of the parameters in the ``neutron-nuage-config.yaml`` and ``nova-nuage-config.yaml`` files.
+    - For AVRS integration, some of the parameters in ``fast-path.env`` needs to be configured in the Heat templates. We use ``compute-avrs-environment.yaml`` environment file to configure these values.
+    - For AVRS integration, see the `Sample Templates`_ section for some probable values of the parameters in the ``compute-avrs-environment.yaml`` file and we also need to create a new AVRS role similar to the upstream Compute role.
+    - For AVRS intergration, we can also create Multiple roles which we can pass different sets of configuration on those AVRS Compute Node. Sample enviroment file can be found `here <../../nuage-tripleo-heat-templates/environments/compute-avrs-multirole-environment.yaml>`_
+
+
+Links to Nuage and OpenStack Resources
+---------------------------------------
+
+* For the Heat templates used by OpenStack director, go to http://git.openstack.org/cgit/openstack/tripleo-heat-templates .
+* For the Puppet manifests, go to http://git.openstack.org/cgit/openstack/tripleo-heat-templates/tree/puppet .
+* For the nuage-puppet-modules RPM (nuage-puppet-modules-5.1.0), go to https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/image-patching .
+* For the script to patch the Overcloud qcow image (nuage_overcloud_full_patch.py), go to https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/image-patching/stopgap-script/nuage_overcloud_full_patch.py .
+* For the Nuage and Puppet modules, go to http://git.openstack.org/cgit/openstack/tripleo-heat-templates/tree/puppet .
+* For the files and script to generate the CMS ID, go to https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/generate-cms-id .
+
+.. Note::  **Please contact Nuage for Nuage Ironic Integration**
```

Comment:
Revise the text as follows.

From
**Please contact Nuage for Nuage Ironic Integration**

To with no bold (not required when you use .. Note::)
Contact Nuage for Nuage Ironic Integration.

Reply:
We used Bold here specifically to make sure user doesn't try Ironic Integration.
If we don't use bold here, the user might oversight this comment as it melds with the document and doesn't stand out.

Reply:
I understand now - OK to keep it as bold.  You could also use the Important tag so that the user knows it is something important. 

.. Important:: Contact Nuage for the Nuage Ironic Integration.

Reply:
done